--- a/tests/test_gps.py
+++ b/tests/test_gps.py
@@ -15,8 +15,13 @@
 
 try:
     import beartype
+    import jaxtyping
 
-    ValidationErrors = (ValueError, beartype.roar.BeartypeCallHintParamViolation)
+    ValidationErrors = (
+        ValueError,
+        beartype.roar.BeartypeCallHintParamViolation,
+        jaxtyping.TypeCheckError,
+    )
 except ImportError:
     ValidationErrors = ValueError
 
